apps/web/src/components/molecules/StudentProgress/StudentProgress.tsx

```diff
@@ -10,29 +10,31 @@ interface IStudentProgress {
 const ProgressCircle: React.FC<IStudentProgressComponent> = ({ year, id }) => (
   <div
     className={`${year <= id ? "border-[#153e75]" : "border-[#36b37e]"}
```

Comment:
I think the code will be a bit more concise if we also use `cn` in this file

Reply:
Done!